reference/conanfile/tools/scons.rst

```diff
@@ -0,0 +1,70 @@
+.. _conan_tools_sconsdeps:
+
+conan.tools.scons
+=================
+
+.. important::
+
+   Some of the features used in this section are still **under development**. While they
+   are recommended and usable, and we will try not to break them in future releases, some
+   breaking changes might still occur.
```

Comment:
We should use this message from a common module to be included anywhere.

Reply:
Yeah, not worth for 1.X

Reply:
Sorry, I did not realize that it's for 1.x.